source/clear-linux/guides/maintenance/download-verify-decompress-linux.rst

```diff
@@ -1,15 +1,15 @@
-.. _download-verify-uncompress-linux:
+.. _download-verify-decompress-linux:
 
-Download, verify, and uncompress a Clear Linux image on Linux
+Download, verify, and decompress a Clear Linux image on Linux
```

Comment:
Can update the "Clear Linux" in page title to use |CL-ATTR| sub. (First use on page)